docs/api-docs/cart-and-checkout/open-source-checkout.md

```diff
@@ -64,7 +64,7 @@ You can upload a custom checkout to your store's server using WebDAV. The instru
 4. Navigate into the `/dist` folder in your checkout project.
 5. Copy the contents of the `/dist` folder and paste it into the checkout folder.
 
-## Installing a custom checkout  
+## <a id="installing-a-custom-checkout"></a> Installing a custom checkout  
```

Comment:
We shouldn't need the `<a>` tag.

I'm pretty sure the reason the anchor isn't working is because the H1 used for the page's title and the H2 used here have the exact time name. Try changing either the H1 or the H2 to something like "Installing custom checkouts" (so its not exactly the same as "Installing a custom checkout").

Reply:
I made the change and I think the anchor works now. Thank you.